data/free worlds start.txt

```diff
@@ -1399,6 +1399,162 @@ mission "FW Pirates 3.1B"
+		system Shaula
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Small Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Small Core Pirates"
+		fleet "Small Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		ship "Bastion (Heavy)" "Dread"
+		ship "Carrier (Alpha)" "Trafficker"
+		ship "Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
+		"Marauder Falcon (Engines)"
```

Comment:
what the f-
please try this mission and tell me your game didn't crash after you lost all fps
and players with 5k combat rating can't possibly handle this; you'd need a FLEET of alien ships with Mereti or Quarg weapons